data/actionsModels.js

```diff
@@ -0,0 +1,36 @@
+const knex = require('knex');
+const dbConfig = require('../knexfile');
+const db = knex(dbConfig.development);
+
+function get(id){
+    if(id){
+        return db('actions').where('id', id).then(action => { return action[0] })
+    }
+
+    return db('actions').then(actions => { return actions })
+}
+
+function add(action){
+    return db('actions').insert(action).then( ([id]) => this.get(id) )
+}
+
+function getActions(id){
+    return db('actions').where('project_id', id).then(actions => {
+            
+        const mapped = actions.map( x => {
+                
+                let newObj = {
+                    id: x.id,
+                    description: x.action_description,
+                    notes: x.action_notes,
+                    completed: x.completed
```

Comment:
Since it looks like you're just taking all of the properties off of each action, you could just do this:
```javascript
let newObj = { ...x }
return newObj
```
which, why use the extra variable in the first place? I don't think it's really needed for readability or anything here:
```javascript
return { ...x }
```
I'd also prefer a more descriptive parameter name for each action, like `action` (instead of `x')